english/movie-ripoffs/hogwarts/hogwarts.md

```diff
@@ -0,0 +1,28 @@
+Your guts turn inside out as your apparation spell throws you violently halfway across the world.
+You look around at the inside of the great hall at Hogwarts school of Witchcraft and Wizardry.  
+It's just after dinner and the place is crowded with students.
+
+A bearded Giant clumps up to you yelling "Cor blimey - it's HARRY POTTER!" as he picks you up in a violent bear hug.
+"I think you're badly mistaken" you exclaim breathlessly as he places you back on the ground.
+
+"No - no mistake, Harry.", he says gleefully as he quickly scribbles a zig-zag on your forehead with a permanent marker pen.  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.